### PR TITLE
Fixed config commitee typo

### PR DESCRIPTION
**Pull request by shravanhariharan2**
open source is my passion

renamed "hack" to "Hack" in config committee since API filters events by committee case-sensitively

### CODE REVIEW

**Comment by shravanhariharan2** [comment]
requesting review for this 1 char change since I'm not sure if anything needs to be manually updated/deployed on netlify after merging

**Review by StormFireFox1 — APPROVED** [review]
As a note for the future, Netlify tracks version control commits and deploys automatically, so you're good to just merge this if you want to.

I'll give you an approval, though, so you get that dopamine hit.

**Pull request merged**
